python/mxnet/gluon/data/dataloader.py

```diff
@@ -214,6 +214,7 @@ def __iter__(self):
             worker.start()
             workers.append(worker)
 
+        idx = 0
```

Comment:
?

Reply:
This is a lint fix

Reply:
This is in a path where pylint check is enforced. Why now?

Reply:
Good question, I don't really know :/ pylint just complained about a few things and I fixed them to get the tests to pass

Reply:
Is it because the pylint version is changing? Should we somehow freeze it?

Reply:
I thought I froze it, but it seems like it's still melting a bit :P would you mind if I address this later?

Reply:
OK